src/SFA.DAS.Roatp.ProviderModeration.Web/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Security.Claims;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Roatp.ProviderModeration.Web.Infrastructure;
+using SFA.DAS.Roatp.ProviderModeration.Web.Models;
+
+namespace SFA.DAS.Roatp.ProviderModeration.Web.Controllers;
+public class ErrorController : Controller
+{
+    private readonly ILogger<ErrorController> _logger;
+
+    public ErrorController(ILogger<ErrorController> logger)
+    {
+        _logger = logger;
+    }
+
+    [AllowAnonymous]
```

Comment:
you can just put [AllowAnonymous] at the top of the controller and remove them from the mehods.  Smoke test it though.   I'll leave it up to you

Reply:
Done